backfill_corrections/Makefile

```diff
@@ -15,12 +26,27 @@ dev: delphiBackfillCorrection_1.0.tar.gz
 lib:
 	R -e 'roxygen2::roxygenise("delphiBackfillCorrection")'
 
-run-R:
-	time Rscript run.R 2>&1 | tee $(LOG_FILE)
+run-local:
+	time Rscript run.R $(OPTIONS) 2>&1 | tee $(LOG_FILE)
 	grep "backfill correction completed successfully" $(LOG_FILE)
 	grep "scheduled core" $(LOG_FILE) ; \
 	[ "$$?" -eq 1 ]
 
+run:
+	# Assumes receiving is output dir
+	docker run --rm --pull=always \
+	    -v "${PWD}"/logs:/backfill_corrections/logs \
+	    -v "${PWD}"/receiving:/backfill_corrections/receiving \
+	    -v "${PWD}"/input:/backfill_corrections/input \
+	    -v "${PWD}"/cache:/backfill_corrections/cache \
+	    -v "${PWD}"/params.json:/backfill_corrections/params.json \
+		-i "${DOCKER_IMAGE}:${DOCKER_TAG}" \
+		/bin/bash -c "make run-local OPTIONS=\"${OPTIONS}\""
+
+deploy: run
+	# Assumes receiving is output dir
+	aws s3 cp receiving/ $(S3_BUCKET)/ --recursive --exclude "*" --include "*csv.gz" --acl public-read
```

Comment:
Does this need credentials installed somehow?

Reply:
Added an `aws configure` step right before this.